lasagne/layers/shape.py

```diff
@@ -98,8 +95,8 @@ def get_output_shape_for(self, input_shape, *args, **kwargs):
                                      (o[0], len(input_shape)))
                 output_shape[dim] = input_shape[o[0]]
                 masked_output_shape[dim] = input_shape[o[0]]
-                if ((input_shape[o[0]] is None)
-                    and (masked_input_shape[o[0]] is None)):
+                if (input_shape[o[0]] is None) \
+                   and (masked_input_shape[o[0]] is None):
```

Comment:
Frankly, I think they're all ugly.  I switched to double indent with line continuation, if you all decide you prefer a different technique, feel free to change it (will push soon).